pmlc/dialect/pxa/ir/ops.td

```diff
@@ -48,8 +48,17 @@ def ImplicitAffineTerminator
 
 def AffineParallelForOp : PXA_Op<"parallel_for", [ImplicitAffineTerminator]> {
   let summary = "multi-index parallel for operation";
-  let arguments = (ins I64ArrayAttr:$ranges, Variadic<Index>:$dynamic_ranges);
-  let regions = (region SizedRegion<1>:$inner);
+  let arguments = (ins AffineMapAttr:$ranges, AffineMapAttr:$transform, Variadic<Index>:$mapOperands);
+  let regions = (region SizedRegion<1>:$region);
+  let extraClassDeclaration = [{
+    ValueRange rangeMapOperands() { return llvm::make_range(operand_begin(), operand_begin() + ranges().getNumInputs()); }
+    ValueRange transformMapOperands() { return llvm::make_range(operand_begin() + ranges().getNumInputs(), operand_end()); }
```

Comment:
I think we should be consistent and use a `get` prefix for all of these.

Reply:
Done.